src/test/scala/uk/gov/nationalarchives/tdr/api/service/ConsignmentStatusServiceSpec.scala

```diff
@@ -102,10 +129,10 @@ class ConsignmentStatusServiceSpec extends AnyFlatSpec with MockitoSugar with Re
 
     val response = consignmentService.getConsignmentStatus(consignmentId).futureValue
 
-    response should be(CurrentStatus(None, None, None))
+    response should be(CurrentStatus(None, None, None, None))
   }
 
-  "setUploadConsignmentStatusValueToComplete" should "update a consignments' status when upload is complete" in {
+  "setUploadConsignmentStatusValueToComplete" should "add the correct series status" in {
```

Comment:
Did you mean to change this one? This method doesn't look like it checks the series.

Reply:
oops I didn't mean to change that